src/modules/opengl/mlt_movit_input.h

```diff
@@ -44,10 +44,10 @@ class MltInput
 
 private:
 	mlt_image_format m_format;
-	unsigned m_width, m_height;
+	unsigned m_width = 1, m_height = 1;
```

Comment:
This valid but a minor consistency complaint: this change mixes initialization of members between declaration and constructor.